Emux.GameBoy/GameBoy.cs

```diff
@@ -114,24 +173,161 @@ public bool IsPoweredOn
             private set;
         }
 
-        /// <summary>
-        /// Resets the state of the GameBoy to the bootup state.
-        /// </summary>
-        public void Reset()
+		public double SpeedFactor => Cpu.CyclesPerSecond / (GameBoyCpu.OfficialClockFrequency * Cpu.SpeedMultiplier);
+
+		private void nextFrame(object sender, EventArgs e)
```

Comment:
Rename to `NextFrame` or perhaps  `ClockOnTick`  since it is an event handler to the tick event of `_clock`.

Reply:
But its private, why UpperCamelCase?

Reply:
PascalCase/UpperCamelCase is the preferred naming convention for methods in C#, regardless of accessibility. This is consistent with the rest of the project, as well as the rest of the .NET framework.

https://www.c-sharpcorner.com/UploadFile/8a67c0/C-Sharp-coding-standards-and-naming-conventions/

Reply:
Fine. If this is the standard you are following then I would add this to the README or make a PR template so anyone can know your style.
This is going to be the biggest challenge now, thinking of this style not the one from work.

Reply:
That is fair, I will add the conventions to the readme when I get the chance.
Excuse my late replies, life has been quite busy unfortunately for me.